src/components/EditCampaign.js

```diff
@@ -310,7 +318,15 @@ class EditCampaign extends React.Component {
             maxAmount : maxAmount,
             currencyName: metaData.currencyName
         });
+        if(metaData.descriptionEditor){
+            setEditorState(metaData.descriptionEditor, true);
+            this.setState({updatedEditorState : true});
```

Comment:
what's the point of switching `updateEditorState` on/off?

Reply:
this was to make sure the data is loaded before the editor component is rendered. can't really update state or re-render the editor once it's there.